scripts/maven-release-publisher.yml

```diff
@@ -1,6 +1,8 @@
 version: 0.2
 env:
   shell: /bin/sh
+  variables:
+    ORG_GRADLE_PROJECT_useAwsSdkReleaseBuild: true
```

Comment:
just to clarify, this means that we only use the release build when publishing Amplify (and snapshot at all other times), right?  also it won't be active for dev-preview until we merge, so do we need to do that by Friday or is the action item just for main?

Reply:
Correct, it should use snapshot for all the build expect release build. this change is just intented for aws android sdk on main branch